### PR TITLE
/import/events route to import JSON file for event debugging

### PR DESCRIPTION
**Pull request by Alex-Tideman**
## What was changed
A new /import/events route was added to support importing JSON files for viewing event history

Screen Shot 2022-03-28 at 11 32 40 AM
Screen Shot 2022-03-28 at 11 33 18 AM

## Why?
This will allow the SDK team and others to import JSON files to view and troubleshoot events

## Checklist

2. How was this tested:
Ran test suite, retested all workflow pages with RainbowStatus workflows to confirm existing pages still all work, added new /import Cypress tests

3. Any docs updates needed?

### CODE REVIEW

File: src/lib/components/event/event-group-details.svelte [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+  import { page } from '$app/stores';
+
+  import EventDetails from '$lib/components/event/event-details.svelte';
+  import Link from '$lib/components/link.svelte';
+
+  export let event: HistoryEventWithId;
+  export let eventGroup;
```
Comment (GiantRobots) [inline src/lib/components/event/event-group-details.svelte:8]: Would be nice to have this typed

File: src/lib/components/event/event-history-import.svelte [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  import { routeForImport } from '$lib/utilities/route-for';
+  import { goto } from '$app/navigation';
+
+  import Button from '$lib/components/button.svelte';
+  import { toEventHistory } from '$lib/models/event-history';
+  import { notifications } from '$lib/stores/notifications';
+  import { importEvents, importEventGroups } from '$lib/stores/import-events';
+  import { faFileImport } from '@fortawesome/free-solid-svg-icons';
+
+  let rawEvents;
```
Comment (GiantRobots) [inline src/lib/components/event/event-history-import.svelte:11]: Type!

Comment (Alex-Tideman) [reply src/lib/components/event/event-history-import.svelte]: Added type: any with a comment

File: src/lib/components/event/event-group-details.svelte (hunk 1 above)
Comment (Alex-Tideman) [reply src/lib/components/event/event-group-details.svelte]: Added the type for this guy

**Review by GiantRobots — APPROVED** [review]

**Pull request merged**